### ISSUE TITLE
Whammy and Overdrive are constantly being activated

### ISSUE THREAD

**Issue by KyleSprawka** [InvoxiPlayGames/FestivalInstrumentMapper#135]
With an Xbox 360 Wired Guitar(X-Plorer) the whammy and overdrive(tilt?) are being activated constantly. As soon as I hit the final note of the Overdrive notes, overdrive activates. Whenever I hit a sustain note, the game acts as if the whammy is being pressed down slowly for each note, until the whammy is completely depressed(audibly, the bar is not being pressed at all).

**Comment by gkazuakix** [comment]
Having the same issue. Using Les Paul Wii Guitar Hero with RCM Adapter.
For me, the overdrive/tilt thing is because I play with the guitar tilted up. If I go into the RCM Programming Tool, I can see that the "tilt" control is constantly trigged if the adapter isn't flat.
The whammy constantly being activated is the real kicker for me. I can see in the Programming Tool that the whammy is calibrated correctly, but I guess Fortnite's engine doesn't have this calibration function. I thought before the Metallica season the whammy was working fine with Festival Mapper, but I might be wrong.
I also posted here but the solutions users tried to give don't work:
[](https://www.reddit.com/r/FortniteFestival/comments/1ew5qmu/autowhammy_how_to_fix/)

**Comment by glitchyre** [comment]
With an Xbox 360 Guitar my whammy bar doesn't input anything or does a constant input is there any way to calibrate it?

**Comment by KatSauce** [comment]
Yeah I'll bump this again cause it REALLY needs to be addressed. I have a 360 Xplorer and it's just plug and play, don't have this issue in CH or YARG or GHWTDE,  there's no config in this mapping tool to mess with it, and it makes the game sound AWFUL having every held note be WILDLY changing tone and unable to hold onto any overdrive unless my guitar is basically flat on the table

**Comment by supersammy123** [comment]
yeah I have the same issue and have to play with my guitar perfectly flat and this is hella uncomfortable,

one solution i found was recalibrating the z axis to be above vertical in the "Game Controller" settings (on windows 10 idk where it is on windows 11)
when it tells you to move it from 0-100, just start it in a comfortable position and end in the same place

This doesn't mean my whammy bar works but I can hold the guitar comfortably without overdrive and whammy constantly activating, its not a fix but its a step there
